### PR TITLE
local: Add bindle release build for better Linux compat

### PR DESCRIPTION
**Pull request by lann**

### CODE REVIEW

**Comment by lann** [comment]
ref #58

**Comment by lann** [comment]
These binaries seem to work. They could just be checked in to the repo, or they could be artifacts on the github release.

**Comment by vdice** [comment]
A thought: If we added Docker as a dep for the local installer, we could run bindle via the container from the Dockerfile in this PR.  WDYT?

**Comment by lann** [comment]
This works for me locally

File: local/job/bindle.nomad [hunk 1]
```diff
@@ -58,7 +58,7 @@ job "bindle" {
       driver = "raw_exec"
 
       artifact {
-        source = "https://bindle.blob.core.windows.net/releases/bindle-v0.8.0-${var.os}-${var.arch}.tar.gz"
+        source = "https://raw.githubusercontent.com/fermyon/installer/9fca2d7b641440cef0f75d38ace9cd3128b5a8a3/local/bindle/bindle-server"
```
Comment (lann) [inline local/job/bindle.nomad:61]: This is commit id of the first commit in this PR.

**Comment by vdice** [comment]
This worked well for me (on Mac) -- so, a test to ensure default behavior is maintained.  I believe @lann also tested on a Linux box to vet the bindle-server URL.  Wondering if @bacongobbler will have a chance to try this out and sign off?

**Comment by lann** [comment]
Would be nice if @itowlson could test too on Ye Olde Linuks

**Review by bacongobbler — APPROVED** [review]
Tested on Ye Olde Linuxe

LGTM

**Comment by itowlson** [comment]
Fails on 18.04 LTS, I'm afraid.

With no diagnostics, which is vexing - our script keeps retrying until timeout, but Nomad itself seems to provide absolutely no logs of process startup errors - surely they must be there somewhere?!  Anyway, it works for me on 20.04 so I _assume_ it's Hamster Stonehenge issues rather than WSL issues, but can't confirm for certain...!

**Comment by lann** [comment]
> Nomad itself seems to provide absolutely no logs of process startup errors - surely they must be there somewhere?!

This incantation seems to work; at least it gives you an exit code...
```bash
$ nomad job status bindle
[...SNIP...]
Allocations
ID        Node ID   Task Group  Version  Desired  Status  Created    Modified
bd840c46  0659ac07  bindle      0        run      failed  1m3s ago   43s ago
e68e94b8  0659ac07  bindle      0        stop     failed  1m49s ago  1m3s ago

$ nomad alloc status bd840c46
[...SNIP...]
Recent Events:
Time                       Type             Description
2022-06-27T11:12:53-04:00  Restarting       Task restarting in 17.645456123s
2022-06-27T11:12:53-04:00  Terminated       Exit Code: 2
2022-06-27T11:12:53-04:00  Started          Task started by client
```

**Comment by itowlson** [comment]
Yeah but I wanted stderr to figure out _why_ it exited with code 2.

**Pull request merged**